PKHeX/Resources/text/fr/lang_fr.txt

```diff
@@ -361,4 +361,14 @@ L_Country = Pays
 - **Please leave the {0} in your line. The OT name will show instead.
 --
 L_Arguments = Onglet désactivé ; Toujours avec ; DO ; Génération précédente - ; Souvenirs avec ; Pokémon  ; Zone  ; Objet(s)  ; Attaque  ; Emplacement 
+! End
+! -----------------------------------------------------
+- DO NOT CHANGE THIS SECTION.
+! ErrorWindow = Erreur
+- Change stuff below this line, not above.
+-------------------------------------------------------
+L_ProvideInfo = S'il vous plaît fournir ces informations lorsque vous rapportez cette error: 
+B_CopyToClipboard = Copier dans les Notes
+B_Continue = Continuer
+B_Abort = Abandonner
```

Comment:
I'd appreciate it if someone more familiar with French could proof-read my translation.